python/pyspark/sql/udf.py

```diff
@@ -35,24 +37,29 @@ def _wrap_function(sc, func, returnType):
                                   sc.pythonVer, broadcast_vars, sc._javaAccumulator)
 
 
+def _get_argspec(f):
```

Comment:
How about putting this in pyspark.util? It might be useful in places other than sql

Reply:
Make sense. Moved to pyspark.util.